core/src/banking_stage/qos_service.rs

```diff
@@ -247,25 +247,25 @@ impl QosService {
         let &BatchedTransactionDetails {
             costs:
                 BatchedTransactionCostDetails {
```

Comment:
was on the fence about these ones since these are costs which COULD potentially overflow.

but considering we just roll them into an AtomicU64 which wraps anyway it seems fine to me